package/autoyast2.spec

```diff
@@ -22,7 +22,7 @@
 %endif
 
 Name:           autoyast2
-Version:        4.0.10
+Version:        4.0.11
```

Comment:
I miss increasing the dependency with yast2-storage-ng to the new version supporting custom callbacks

Reply:
Strictly speaking, is yast2-installation who uses such a feature. But I will update this one aswell.